content/docs/1.5.3/v2-data-engine/features.md

```diff
@@ -1,6 +1,8 @@
 ---
 title: Features
 weight: 1
+aliases:
+- /spdk/features.md
```

Comment:
It seems https://deploy-preview-855--longhornio.netlify.app/docs/1.5.3/spdk/features not working?

Reply:
I assume that the redirects will work after the aliases are actually published. Since this PR has not been merged and the 'spdk' folder no longer exists, any links containing 'spdk' will be broken.

Reply:
Ok, we can see how it will work after the merge.